Documentation/GettingStarted.md

```diff
@@ -23,16 +27,77 @@ import AEPSignal
 2. Register the core extensions, configure the core with your assigned application identifier, and start lifecycle metrics.
 To do this, you add the following code to your Application Delegate's `application(_:didFinishLaunchingWithOptions:)` method:
```

Comment:
we can remove lifecycle related words and code from this step, since now we have a new segment for it